app/templates/orders/_summary_tabs.hbs

```diff
@@ -0,0 +1,19 @@
+<section class="main-section order_item_list_tabs">
+  <div class="row">
+    <div class="small-12 columns tab_row">
+      <dl class="tabs" data-tab>
+        {{#link-to 'orders.contact_summary' model.id tagName="dd" classNames="small-6 medium-6 columns text-center"}}
+          {{#link-to 'orders.contact_summary' model.id}}
+            <div>Contact</div>
```

Comment:
btw, Matt asked me to go through Stock and make sure everything is in the translation file. It'll take a while, but if you find any strings hardcoded into the templates, make sure to move them to the localisation files 😄

Reply:
I'm still not used to adding translations in stock app. 😄 
I'll make the chagnes

Reply:
Same, doing that now myself